mesheryctl/cmd/update.go

```diff
@@ -15,8 +15,15 @@
 package cmd
 
 import (
+	"encoding/json"
+	"fmt"
+	"github.com/artdarek/go-unzip"
```

Comment:
@anandrkskd PR looks great 👏 
One request: can we please move away from this go-unzip library and use the [archive/zip](https://golang.org/pkg/archive/zip/) package.
Here is an article with an example: https://golangcode.com/unzip-files-in-go/

Reply:
Understood!